blackjax/inference/trajectory.py

```diff
@@ -275,6 +313,162 @@ def add_one_state(expansion_state):
     return integrate
 
 
+def dynamic_recursive_integration(
+    integrator: Callable,
+    kinetic_energy: Callable,
+    uturn_check_fn: Callable,
+    divergence_threshold: float,
+    use_robust_uturn_check: bool = False,
```

Comment:
Do you see any scenario where we might want to turn this off?

Reply:
Setting it False here to make sure the progressive implementation is equivalent to the recursive. Just for clarity the recursive implementation should not be used for serious work as it cannot be jitted and thus likely slow.